node/src/main/scala/coop/rchain/node/effects/ReplClient.scala

```diff
@@ -70,7 +66,7 @@ class GrpcReplClient[F[_]: Monixable: Sync](host: String, port: Int, maxMessageS
   }
 
   private def readContent(filePath: Path): String =
-    Resources.withResource(Source.fromFile(filePath.toFile))(_.mkString)
+    Using.resource(Source.fromFile(filePath.toFile))(_.mkString)
```

Comment:
In some cases you are using `Using` but here is `Using.resource`. What is difference? With `Using` we have ability to handle exceptions?

Reply:
> If you wish to avoid wrapping management and operations in a Try, you can use Using.resource, which throws any exceptions that occur.
https://www.scala-lang.org/api/2.13.x/scala/util/Using$.html

`Using.resource` replicates existing behavior to throw exception.